<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDlg</class>
 <widget class="QDialog" name="addDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>35</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Level:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="levelLE">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>35</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nameCB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="minimumContentsLength">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>35</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="typeLE">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>35</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Version:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="versionLE">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>75</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Job:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="jobCB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>骑士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>战士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>暗黑骑士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>绝枪战士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>武僧</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>龙骑士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>忍者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>武士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>钐镰客</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>吟游诗人</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>机工士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>舞者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>黑魔法师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>召唤师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>赤魔法师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>白魔法师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>学者</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>占星术士</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>贤者</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>75</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>75</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluate:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="evaluateCB">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一帆风顺</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>有人倒了</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>团灭了</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>我是稿子</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>T不会</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>N不会</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>D不会</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>608</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>676</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
